web/src/services/api/api-client.ts

```diff
@@ -1,4 +1,5 @@
 class ApiClient {
+  static base_url = process.env.REACT_APP_API_BASE_URL;
```

Comment:
In production, the base URL will be dynamic. (ie) users can change the port of their choice while starting the appium server. So we have to keeo it empty just for prod env.

Reply:
So currently we're reading this from the env file. we can change the env file accordingly during the build stage itself. thoughts?

Reply:
Sounds good. Then could you add `export REACT_APP_API_BASE_URL=/dashboard` on top of https://github.com/sudharsan-selvaraj/appium-dashboard-plugin/blob/master/scripts/build_web.sh#L1 file so that during publish we will make sure the API URL is loaded properly. In future, we will handle this via https://github.com/sudharsan-selvaraj/appium-dashboard-plugin/issues/17

Also, could you replace npm with yarn in https://github.com/sudharsan-selvaraj/appium-dashboard-plugin/blob/master/scripts/build_web.sh and go ahead merging the changes.